### DISCUSSION TITLE
mongo bd en windows 8.1 pro

### DISCUSSION THREAD

**Discussion by igmroman** [Launch-X-Latam/MisionFrontEnd#545] [Team Posts]
![mongo](https://user-images.githubusercontent.com/18242463/201203545-f9648019-f9a6-4473-88f6-4e45026413ac.JPG)

intente instalar mongodb sin embargo me sale esta ventana, agradeceria alguien me dijera
si hay alguna opción alternativa o ¿deberia instalar el sistema operativo windows 10?

**Comment by SantiagoCastellani** [comment]
Buenas. 
Intentá con una versión anterior. recomienda la 4.4. Quizás una menor.
